site/content/en/latest/user/http-routing.md

```diff
@@ -129,6 +129,98 @@ curl -vvv --header "Host: bar.example.com" --header "env: canary" "http://${GATE
 A `200` status code should be returned and the body should include `"pod": "bar-canary-backend-*"` indicating the
 traffic was routed to the foo backend service.
 
+### JWT Claims Base Routing
+
+Users would like to route to a specific backend by matching on JWT claims, users can achieve this base on [JWT Authentication](./jwt-authentication.md) configuration.
+
```

Comment:
```suggestion
This can be achieved, by defining a SecurityPolicy with a jwt configuration that does the following
* Converts jwt claims to headers, which can be used for header based routing
* Sets the recomputeRoute field to `true`. This is required so that the incoming request matches on a fallback/catch all route where the JWT can be authenticated, the claims from the JWT can be converted to headers, and then the route match can be recomputed to match based on the updated headers.

For this feature to workplease make sure
* you have a fallback route rule defined, the backend for this route rule can be invalid.
* The SecurityPolicy is applied to both the fallback route as well as the route with the claim header matches, to avoid spoofing.

```